cli_test/cli_posts_test.go

```diff
@@ -78,7 +78,7 @@ func TestDesmosCLIPostsCreateWithMedias(t *testing.T) {
 
 	// Later usage variables
 	subspace := "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e"
-	message := "message"
+	message := ""
```

Comment:
This should not be done. Instead, a new case should be added. We need to test all cases, not just one of them: 

- empty medias, non-empty message
- non-empty medias, non-empty message
- non-empty medias, empty message